src/App.tsx

```diff
@@ -1,11 +1,19 @@
-import { useState } from 'react';
+import { useCallback, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const handleAddMovie = useCallback((movie: Movie) => {
+    const isPresent = movies.some(m => m.imdbId === movie.imdbId);
+
+    if (!isPresent) {
```

Comment:
make sense to show message to user something like: `This movie already in the list`